crocket/src/main/java/com/crocket/Ball.java

```diff
@@ -0,0 +1,30 @@
+package com.crocket;
+public class Ball extends Entity implements IMovable{
+    private double mass;
+    private String powerUp;
+    private double xVelocity;
+    private double yVelocity;
+
+    public Ball(int width, int height, double xPosition, double yPosition, double mass) {
+        super(width, height, xPosition, yPosition);
+        //TODO Auto-generated constructor stub
+    }
+
+    public double getxVelocity(){
+        return xVelocity;
+    }
+
+    public double getyVelocity(){
+        return yVelocity;
+    }
```

Comment:
what do we need getVelocity for?

Reply:
For now, nothing but it will be needed to implement collision. I will add these methods to the IMovable interface in a later pull request for collision